tests/pubsub.rs

```diff
@@ -39,13 +39,14 @@ async fn unsubscribe_via_drop() {
     assert_eq!(a.pubsub_subscribed().await.unwrap(), empty);
 }
 
-#[tokio::test]
-async fn can_publish_without_subscribing() {
-    let a = Node::new("test_node").await;
-    a.pubsub_publish("topic".into(), b"foobar".to_vec())
-        .await
-        .unwrap()
-}
+// This test may not be needed since rust-libp2p gossip implementation doesnt allow publishing without first subscribing(?)
+// #[tokio::test]
+// async fn cant_publish_without_subscribing() {
```

Comment:
Commented this out since gossip doesnt support publishing without subscribing. A topic would need to be subscribed and would also need another peer available in the topic for function to succeed.